src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js

```diff
@@ -105,5 +118,8 @@ export default function PackageConfirmation({
 PackageConfirmation.propTypes = {
   packageConfirmation: PropTypes.shape({
     confirmationPopup: propTypes.confirmationPopup
+  }).isRequired,
+  csoAccountStatus: PropTypes.shape({
+    isNew: PropTypes.bool
```

Comment:
no `exists` property here?

Reply:
Should we pass the exists? Currently, the only thing we check is if the account is new, not if the account exists

Reply:
hmm I see, yeah I guess the assumption is that when we arrive on this page, cso account already exists.
so no need to pass in, cool